plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/scanResults/IgnoreDetails.java

```diff
@@ -0,0 +1,44 @@
+package io.snyk.languageserver.protocolextension.messageObjects.scanResults;
+
+public class IgnoreDetails {
```

Comment:
I think this class needs `equals` and `hashCode` methods to be able to sort & compare them